### ISSUE TITLE
[FEATURE] Add support of cattr 22.2.0

### ISSUE THREAD

**Issue by kemics** [Toloka/toloka-kit#161]
### Problem description

Currently when I'm trying to install package to Google Cloud Composer (managed Airflow cluster) I get the following error on recent version of Composer:

`toloka-kit 1.0.2 has requirement cattrs<22.2.0,>=1.1.1, but you have cattrs 22.2.0.`

I can see in [setup.py](https://github.com/Toloka/toloka-kit/blob/4d338a9d6f142c2a86b045b611db9e1003bf1959/setup.py) a comment:
```
# cattr 22.2.0 breaks tests/primitives/test_operator.py
        'cattrs >= 1.1.1, < 22.2.0',
```

### Feature description

Can you please consider adding support of `cattr==22.2.0`?

**Comment by Pocoder** [comment]
@kemics
Hi, thank you for the issue.
We will investigate why new version of cattr breaks tests and will try to fix it.

**Comment by kemics** [comment]
@Pocoder I see that the MR was merged, thank you for that, incredibly fast 👍 

I would appreciate if v1.1.3 will be released soon. It will help in my case to upgrade the Airflow cluster sooner

**Comment by Pocoder** [comment]
@kemics The issue was fixed and released in version 1.1.3.
